tests/dummy/app/templates/application.hbs

```diff
@@ -2,14 +2,47 @@
 
 <h2 id="title">Welcome to @ember-intl/cp-validations</h2>
 
-<div class="email-validation">{{model.validations.attrs.email.message}}</div>
-<div class="emailConfirmation-validation">{{model.validations.attrs.emailConfirmation.message}}</div>
-<div class="password-validation">{{model.validations.attrs.password.message}}</div>
-<div class="passwordConfirmation-validation">{{model.validations.attrs.passwordConfirmation.message}}</div>
-<div class="username-validation">{{model.validations.attrs.username.message}}</div>
-
-{{input id="email" value=model.email}}
-{{input id="emailConfirmation" value=model.emailConfirmation}}
-{{input id="password" value=model.password}}
-{{input id="passwordConfirmation" value=model.passwordConfirmation}}
-{{input id="username" value=model.username}}
+<input
+  id="email"
+  value={{this.model.email}}
```

Comment:
**suggestion (optional)**

For accessibility, I'd recommend specifying the `type`. For example, `type="email"`, `type="password"`, and `type="text"`. Specifying the type can also help us avoid re-implementing basic validations.

Reply:
https://github.com/ember-intl/cp-validations/pull/90/commits/131657a7eb4e28266461c4e674ffa91ac94a8380